layersvt/api_dump.h

```diff
@@ -515,22 +520,23 @@ class ApiDumpSettings {
         }
     }
 
-    inline ApiDumpFormat format() const { return output_format; }
+    ApiDumpFormat format() const { return output_format; }
 
-    std::ostream &formatNameType(std::ostream &stream, int indents, const char *name, const char *type) const {
+    void formatNameType(std::ostream &stream, int indents, const char *name, const char *type) const {
         stream << indentation(indents) << name << ": ";
-
         if (use_spaces)
-            stream << std::setfill(' ') << std::setw(name_size - (int)strlen(name) - 2);
+            stream << std::setw(name_size - (int)strlen(name) - 2) << "";
```

Comment:
Its a case of `std::setw()` not actually 'applying' when the semicolon is reached. 
This should have a comment because I realize how weird that is to see.